src/main/java/org/gridsuite/securityanalysis/server/service/SecurityAnalysisWorkerService.java

```diff
@@ -133,87 +140,103 @@ private static String sanitizeParam(String param) {
 
         LOGGER.info("Run security analysis on contingency lists: {}", context.getContingencyListNames().stream().map(SecurityAnalysisWorkerService::sanitizeParam).collect(Collectors.toList()));
 
-        Mono<Network> network = getNetwork(context.getNetworkUuid(), context.getOtherNetworkUuids());
+        if (resultUuid != null && cancelComputationRequests.get(resultUuid) != null) {
+            return Mono.empty();
+        }
+
+        LOGGER.info("Loading networks");
```

Comment:
The logs should happen when the operation is performed, not when the mono is created. There is the same problem for other log statements in this code, we should fix them all

Reply:
Done